packages/react/src/file-upload/FileUpload.tsx

```diff
@@ -197,129 +239,185 @@ export const FileUpload = forwardRef<HTMLInputElement, FileUploadProps>(
-									css={visuallyHiddenStyles}
-								/>
+						<>
+							<Box {...dropzoneProps}>
 								<Flex
-									flexDirection="column"
+									gap={1}
+									padding={1.5}
 									alignItems="center"
-									textAlign="center"
+									flexDirection="column"
+									border
+									rounded
+									css={styles}
 								>
-									<Text
-										color={isDragActive ? 'action' : 'muted'}
-										fontWeight="bold"
+									<UploadIcon size="lg" color="muted" />
+									<input
+										{...dropzoneInputProps}
+										{...a11yProps}
+										{...consumerProps}
+										/**
+										 * Dropzone needs to set a ref to the input, but we _also_
+										 * need to forward a ref to the input so consumers can use it.
+										 * The mergeRef utility allows us to do this.
+										 */
+										ref={mergeRefs([forwardedRef, dropzoneInputRef])}
+										css={visuallyHiddenStyles}
+									/>
+									<Flex
+										flexDirection="column"
+										alignItems="center"
+										textAlign="center"
 									>
-										{isDragActive
-											? `Drop ${filesPlural} here`
-											: `Drag and drop ${filesPlural} here or select ${filesPlural} to upload.`}
-										{isDragActive ? <>&hellip;</> : null}
-									</Text>
-									{maxSize ? (
-										<Text color="muted">
-											{multiple ? 'Each file' : 'File'} cannot exceed{' '}
-											{formattedMaxFileSize}.
-										</Text>
-									) : null}
-									{accept ? (
-										<Text color="muted">
-											Files accepted: {acceptedFilesSummary}
-										</Text>
-									) : null}
+										{/* This maintains the space created by the default text so when a user drags over the dropzone, its height doesn't change, which can ruin the drag. We then vertically centre the isDragActive text */}
+										<span css={{ position: 'relative' }}>
+											<Text
+												color="muted"
+												css={{
+													visibility: isDragActive ? 'hidden' : undefined,
+												}}
+												fontWeight="bold"
+											>
+												{multiple
+													? 'Drag and drop files here or select files to upload.'
+													: 'Drag and drop a file or select a file to upload.'}
+											</Text>
+											<Text
+												css={{
+													display: isDragActive ? 'block' : 'none',
+													left: 0,
+													position: 'absolute',
+													right: 0,
+													top: '50%',
+													transform: 'translateY(-50%)',
+												}}
+												color="action"
+												fontWeight="bold"
+											>
+												Drop {filesPlural} here…
+											</Text>
+										</span>
+										{maxSize ? (
+											<Text color="muted">
+												{multiple ? 'Each file' : 'File'} cannot exceed{' '}
+												{formattedMaxFileSize}.
+											</Text>
+										) : null}
+										{accept ? (
+											<Text color="muted">
+												Files accepted: {acceptedFilesSummary}
```

Comment:
Should this end a full stop as the `maxSize` message above does?

Reply:
It's not in the design, but perhaps it should. This hasn't changed

Reply:
Done